### DISCUSSION TITLE
Force pagination: how to setCurrent page to 1

### DISCUSSION THREAD

**Discussion by v4voloshyn** [refinedev/refine#4687] [Q&A] [answered]
Hi! I'm lookin for some help with my issue. 
I'm use refine version 3.18.0. And i have a table with tabs above that are basically just sorting the data by status (All, completed, archived etc). Sorting and filtering are put in URLSearchParams as like currentPage when i using pagination buttons. But when i switching the tabs i want to reset currentPage to the first in URLParams, - that pagination buttons do by default, but just switching the tab isn't. I've tried so many times to figure it out how to do it, but can't. You are my last hope, guys! If you need some other input data - feel free to ask here 🙏🏻

**Comment by salihozdemir** [comment, marked as answer]
Hi @v4voloshyn,

I think you can use the `setCurrent` method to set the current 1 when you switch the tabs. Here is an example:

```ts
<Tabs
  onChange={() => {
    setCurrent(1);
  }}
></Tabs>
```

I hope this can help you.
